app/models/message.rb

```diff
@@ -59,4 +55,14 @@ def chat_message_link
       anchor: "chat-row-#{id}"
     )
   end
+
+  private
+
+  def send_outbound_messages
+    return if manually_created? || reply?
```

Comment:
I moved the conditional to the private method because there was a failing test `but if contributor blocked the telegram bot` where the method wasn't being called to send outbound messages when the message was created.

Also, I read that it is a best practice to have the conditionals in the method call to allow them to become more complex and not be defined in the line that creates the `after_commit`. Note that the same place I read this actually suggests to have the guard clause in the worker itself because even though it will enqueue more jobs, they are no ops. See [Rails Callbacks: 5 Best Practices Used at Gusto](https://engineering.gusto.com/the-rails-callbacks-best-practices-used-at-gusto/)

I still however believe that we don't even want to try to send an outgoing message after receiving an incoming one for reasons stated in the issue.